src/ggrc/models/mixins/customattributable.py

```
@@ -184,35 +195,57 @@ def _add_ca_value_dicts(self, values):
       values: List of dictionaries that represent custom attribute values.
     """
     from ggrc.models.custom_attribute_value import CustomAttributeValue
+    from ggrc.models.person import Person
 
     for value in values:
-      if not value.get("attribute_object_id"):
-        # value.get("attribute_object", {}).get("id") won't help because
-        # value["attribute_object"] can be None
-        value["attribute_object_id"] = (value["attribute_object"].get("id") if
-                                        value.get("attribute_object") else
-                                        None)
+      if not value.get("custom_attribute_id"):
+        if "href" in value:
+          # Ignore setting of custom attribute stubs. Getting here
+          # means that the front-end is not using the API correctly
+          # and needs to be updated.
+          logger.info("Ignoring post/put of custom attribute stubs.")
```

Is the frontend doing this often? If so I think we should make a ticket to fix. If not I think we should throw an exception to make sure the frontend never starts doing this.